xknx/management/procedures.py

```diff
@@ -58,3 +66,115 @@ async def nm_individual_address_check(
         # if Disconnect is received immediately, IA is occupied
         logger.debug("Device does not support transport layer connections. %s", ex)
         return True
+
+
+async def nm_individual_address_read(
+    xknx: XKNX, timeout: float | None = 3, raise_if_multiple: bool = False
+) -> list[IndividualAddress]:
+    """
+    Request individual addresses of all devices that are in programming mode. If multiple devices are in programming mode, a ManagementConnectionError is raised.
+
+    :param: timeout specifies the timeout in seconds, the KNX specification requires a timeout of 3s.
```

Comment:
We should document all or none arguments with `:param:`. `raise_if_multiple` isn't.
It doesn't raise by default, so the first paragraph is not appropriate.